### PR TITLE
Update to icon tags for accessibility

### PR DESCRIPTION
**Pull request by alisan617**
[TNL-5849](https://openedx.atlassian.net/browse/TNL-5849)

## Description
- Change i to span for icon

## Reviewers
- [ ] @andy-armstrong 
- [x] @bjacobel

### CODE REVIEW

File: edx_proctoring/static/proctoring/spec/proctored_exam_attempt_spec.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ describe('ProctoredExamAttemptView', function () {
         '<div class="search-attempts">' +
         '<input type="text" id="search_attempt_id" placeholder="e.g johndoe or john.doe@gmail.com"' +
         '<% if (inSearchMode) { %> value="<%= searchText %>" <%} %>' +
-        '/> <span class="search"><i class="fa fa-search"></i></span> <span class="clear-search"><i class="fa fa-remove"></i></i></span>' +
+        '/> <span class="search"><span class="fa fa-search" aria-hidden="true"></span></span> <span class="clear-search"><span class="fa fa-remove" aria-hidden="true"></span></i></span>' +
```
Comment (andy-armstrong) [inline edx_proctoring/static/proctoring/spec/proctored_exam_attempt_spec.js]: There's a stray closing ``</i>`` tag here.

Comment (andy-armstrong) [inline edx_proctoring/static/proctoring/spec/proctored_exam_attempt_spec.js]: Do you need to add the ``icon`` class too? This is the recommendation from the pattern library doc:

    <span class="icon fa fa-info-circle" aria-hidden="true"></span>

File: edx_proctoring/static/proctoring/templates/student-proctored-exam-attempts.underscore [hunk 2]
```diff
@@ -8,8 +8,8 @@
                         value="<%= searchText %>"
                     <%} %>
                     />
-                    <span class="search"><i class="fa fa-search"></i></span>
-                    <span class="clear-search"><i class="fa fa-remove"></i></i></span>
+                    <span class="search"><span class="fa fa-search" aria-hidden="true"></span></span>
```
Comment (andy-armstrong) [inline edx_proctoring/static/proctoring/templates/student-proctored-exam-attempts.underscore]: Same question about ``icon`` class here too.

**Comment by alisan617** [comment]
@andy-armstrong good catch. icon class in our internal class, not required. But I think it's good to be consistent since we use it everywhere.

**Comment by alisan617** [comment]
@andy-armstrong forgot to mention that your requested changes have been made.

**Pull request merged**